src/js/actions/SelectionsActions.js

```diff
@@ -155,7 +156,8 @@ function validateSelectionsForUnloadedTools(projectSaveLocation, bibleId, chapte
       const contextId = selectionsData.contextId;
       if (contextId) {
         if (currentTool !== contextId.tool) {
-          const key = contextId.groupId + ":" + contextId.occurrence + ":" + contextId.quote;
+          const contextIdQuote = Array.isArray(contextId.quote) ? contextId.quote.map(({word}) => word).join(' ') : contextId.quote;
```

Comment:
I don't think this would support punctuation.

Reply:
There are quote arrays that contain punctuation

Reply:
All the keys for the content of the text are in the `word` property at the moment https://github.com/translationCoreApps/tsv-groupdata-parser/blob/master/src/helpers/wordOccurrenceHelpers.js#L107

If you check the resources it is like that as well, `/Users/{user}/translationCore/resources/en/translationHelps/translationNotes/v15/culture/groups/1co/figs-explicit.json`

Reply:
My point is that you're going to join a space before a comma. For example:
```js
[
{
word: "hello",
occurrence: 1
},
{
word: ",",
occurrence: 1
},
{
word: "my",
occurrence: 1
},
{
word: "name",
occurrence: 1
},
{
word: "is",
occurrence: 1
},
{
word: "Manny",
occurrence: 1
},
]
```
This will look like this:

`Hello , my name is Manny`

Reply:
Instead of the correct way which is: 
`Hello, my name is Manny`

Reply:
Yes that is true. The string being made is not shown to the user, it's just a way to distinguish different contextIds with a unique string. Check the next couple lines the key is used to later reference that object

Reply:
If you say is good then its good but hopefully it doesn't cause problems in the future